packages/react/src/components/baseCheckout/index.tsx

```diff
@@ -213,9 +237,29 @@ const BaseCheckout = ({
                       setFieldValue={setFieldValue}
                     />
 
-                    <Button type="submit" disabled={payButtonDisable}>
+                    <ButtonWithTooltip
+                      type="submit"
+                      disabled={
+                        payButtonDisable ||
+                        !getIsBalanceEnough(
```

Comment:
boolean function naming is better with is or has prefixes. let's have isBalanceEnough here

Reply:
isBalanceEnough hints that the variable is a value, not a function. get shows that it is a function which returns isBalanceEnough